src/umpire/ResourceManager.inl

```diff
@@ -58,12 +62,14 @@ Allocator ResourceManager::makeAllocator(
   }
 
   if (!introspection) {
```

Comment:
We could also add a "zero" bool to this method, and optionally opt-out of ZeroTracking.

Reply:
Does our zero tracking have compile guards?

Reply:
Nope

Reply:
I think it would be prudent to add some and leave zero tracking enabled by default.  If you don't agree, that's fine.  Just say so and I'll resolve this conversation.

Reply:
I agree now that we don't need the guards.